app/helpers/service_calendar_helper.rb

```diff
@@ -87,11 +81,6 @@ def display_total_direct_cost_per_study_pppvs service_request
     currency_converter sum
   end
 
-  def display_total_indirect_cost_per_study_pppvs service_request
```

Comment:
Isn't this being used at `app/views/service_requests/calendar_totals.html.haml:44`?